httpserver/https_client.cpp

```diff
@@ -84,6 +110,19 @@ void HttpsClientSession::on_write(error_code ec, size_t bytes_transferred) {
                                std::placeholders::_1, std::placeholders::_2));
 }
 
+bool HttpsClientSession::verify_signature() {
+    const auto it = res_.find(LOKI_SNODE_SIGNATURE_HEADER);
+    if (it == res_.end()) {
+        BOOST_LOG_TRIVIAL(warning)
```

Comment:
We are using `LOKI_LOG` now. (the same below)

Reply:
Oops missed that. Thanks.